scripts/chapter7/7-3_react_search.py

```diff
@@ -151,7 +151,7 @@ def main(query: Optional[str] = None, model: str = "gpt-4o-mini", stop_sequence:
 
     print("\n" + "=" * 60)
     print("最終回答:\n")
-    print(result["output"])
+    print(result.get("output", ""))
```

Comment:
キーが間違っていたら例外を投げさせてもいいですが、ここはログのため、フォールバックさせた方が良いので get を使っています。

https://docs.python.org/3.13/library/stdtypes.html#mapping-types-dict

>d[key]
Return the item of d with key key. Raises a [KeyError](https://docs.python.org/3.13/library/exceptions.html#KeyError) if key is not in the map.

>get(key, default=None, /)
Return the value for key if key is in the dictionary, else default. If default is not given, it defaults to None, so that this method never raises a [KeyError](https://docs.python.org/3.13/library/exceptions.html#KeyError).